### PR TITLE
Download s3 files before evaluation

### PR DESCRIPTION
**Pull request by annehaley**
Relates to #343. I'll merge and test on the prod deployment before closing that issue.

### CODE REVIEW

File: miqa/core/tasks.py [hunk 1]
```diff
@@ -46,34 +55,37 @@ def evaluate_data(frames_by_project):
...
+        for model_name, frame_set in model_to_frames_map.items():
+            current_model = available_evaluation_models[model_name].load()
+            file_paths = []
+            for frame in frame_set:
+                file_path = str(frame.raw_path)
+                if file_path.startswith('s3://'):
+                    tmp = tempfile.NamedTemporaryFile(prefix=tmpdirname)
+                    tmp.write(_download_from_s3(file_path))
+                    file_path = tmp.name
+                else:
```
Comment (dzenanz) [inline miqa/core/tasks.py]: There should be no else. Otherwise we need something like `file_paths.append(tmp.name)` on line 74.

Comment (annehaley) [reply miqa/core/tasks.py]: Nice catch. Changed in [79415e6](https://github.com/OpenImaging/miqa/pull/363/commits/79415e6e8bd6f7d73c44cf05fc17152e5bbeb118)

File: miqa/core/tasks.py [hunk 2]
```diff
@@ -72,8 +72,7 @@ def evaluate_data(frames_by_project):
                     tmp = tempfile.NamedTemporaryFile(prefix=tmpdirname)
                     tmp.write(_download_from_s3(file_path))
                     file_path = tmp.name
```
Comment (dzenanz) [inline miqa/core/tasks.py]: Do we need to update `results=results[str(frame.raw_path)]` on line 83?

Comment (annehaley) [reply miqa/core/tasks.py]: You're right. would something like [20e5673](https://github.com/OpenImaging/miqa/pull/363/commits/20e567360592c3a4f2e2af28946f4539fd3f95b5) work?

**Review by dzenanz — APPROVED** [review]
Preferably squash before or upon merging.

**Pull request merged**